scripts/load_templates.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${DIR}"/.. || exit
+
+DESIGN_SYSTEM_VERSION="13.8.0"
+
+TEMP_DIR=$(mktemp -d)
```

Comment:
If this needs to run before the service is usable, will this work in CF?  I'm not sure if you've got access to a filesystem in it... 
I don't actually know myself, just something that might be an issue?

Reply:
I updated the docker file and it appears to build ok locally now.